docs/docs/overview/guides/aws.mdx

```diff
@@ -0,0 +1,241 @@
+      }}
+    />
+	<span style={{color: "white"}}>Click **Create Project** and create a new project.</span>
+  </TabItem>
+  <TabItem value="step2" label="2">
+    <ThemedImage
+      alt="2"
+      customProps={{
+        zoom: true,
+      }}
+      sources={{
+        light: "/img/guides/aws/step3.png",
+        dark: "/img/guides/aws/step3.png",
+      }}
+    />
+	<span style={{color: "white"}}>Click **Create Capsule** and create a new capsule.</span>
+  </TabItem>
+  <TabItem value="step3" label="3">
+    <ThemedImage
+      alt="3"
+      customProps={{
+        zoom: true,
+      }}
+      sources={{
+        light: "/img/guides/aws/step4.png",
+        dark: "/img/guides/aws/step4.png",
+      }}
+    />
+	<span style={{color: "white"}}>Click on the capsule, then on the **prod** environment.</span>
+  </TabItem>
+  <TabItem value="step4" label="4">
+    <ThemedImage
+      alt="4"
+      customProps={{
+        zoom: true,
+      }}
+      sources={{
+        light: "/img/guides/aws/step5.png",
+        dark: "/img/guides/aws/step5.png",
+      }}
+    />
+	<span style={{color: "white"}}>From the overview page of the Capsule, click **Configure**.</span>
+  </TabItem>
+  <TabItem value="step5" label="5">
+    <ThemedImage
+      alt="5"
+      customProps={{
+        zoom: true,
+      }}
+      sources={{
+        light: "/img/guides/aws/step6.png",
+        dark: "/img/guides/aws/step6.png",
+      }}
+    />
+	<span style={{color: "white"}}>Click on **Images** and type in **nginx** to deploy an **nginx** container.</span>
+  </TabItem>
+  <TabItem value="step6" label="6">
+    <ThemedImage
+      alt="6"
+      customProps={{
+        zoom: true,
+      }}
+      sources={{
+        light: "/img/guides/aws/step7.png",
+        dark: "/img/guides/aws/step7.png",
+      }}
+    />
+	<span style={{color: "white"}}>Click on **Network**, add a new interface and give it a name and expose port 80 (which nginx listens on).</span>
+  </TabItem>
+  <TabItem value="step7" label="7">
+    <ThemedImage
+      alt="7"
+      customProps={{
+        zoom: true,
+      }}
+      sources={{
+        light: "/img/guides/aws/step8.png",
+        dark: "/img/guides/aws/step8.png",
+      }}
+    />
+	<span style={{color: "white"}}>Click **Add host** to make the interface publicly exposed. The host should have an empty configuration. Then click **Deploy**.</span> 
+  </TabItem>
+  <TabItem value="step8" label="8">
+    <ThemedImage
+      alt="8"
+      customProps={{
+        zoom: true,
+      }}
+      sources={{
+        light: "/img/guides/aws/step9.png",
+        dark: "/img/guides/aws/step9.png",
+      }}
+    />
+	<span style={{color: "white"}}>After deploying, wait a few seconds until the rollout shows **All instances are running**.</span>
+  </TabItem>
+  <TabItem value="step9" label="9">
+    <ThemedImage
+      alt="9"
+      customProps={{
+        zoom: true,
+      }}
+      sources={{
+        light: "/img/guides/aws/step10.png",
```

Comment:
Perhaps an image here from the AWS control panel, showing the Load Balancer in question